### DISCUSSION TITLE
I can't build a release

### DISCUSSION THREAD

**Discussion by semilore8** [html2app/discussions#6] [General]
I can't build a release app I have tried more than 10 times but failed but.
    Your app supposed to have the details of what is going on, not to be throwing failed always.
       Maybe you should do a YouTube video on how to properly build a release app

**Comment by yandeu** [comment]
Have you added your credentials? See: 
https://html2app.dev/credentials

Yes, I will make a video.

**Reply by yandeu** [reply]
I have just added some error messages, hopefully yours in included.

**Comment by semilore8** [comment]
Yes I would be looking forward to it on your YouTube channel

**Reply by yandeu** [reply]
Here is the promised video:
https://youtu.be/p0hWpMBmI5c

**Comment by semilore8** [comment]
Yes I added 4 credentials
      ---none of the worked and I added it to the config.js
   But none of them worked.

**Reply by yandeu** [reply]
I will check it.

**Comment by semilore8** [comment]
Ok I will check it out

**Comment by semilore8** [comment]
Got it, thanks!

**Reply by yandeu** [reply]
You're welcome :)
